tests/integration_tests/error/go.mod

```diff
@@ -1,11 +1,11 @@
 module github.com/DataDog/datadog-lambda-go/tests/integration_tests/bin/error
 
-go 1.12
+go 1.13
 
 require (
 	github.com/DataDog/datadog-lambda-go v0.7.0
 	github.com/aws/aws-lambda-go v1.11.1
-	gopkg.in/DataDog/dd-trace-go.v1 v1.27.0
+	gopkg.in/DataDog/dd-trace-go.v1 v1.30.0
```

Comment:
Do you need to commit `integration_tests/error/go.sum`?

Reply:
Probably not, I'll make a `.gitignore` for it

Reply:
Actually, the official recommendation is to commit this, so I think I'll just leave it.